regression/contracts/test_multifile_replace/main.c

```diff
@@ -0,0 +1,20 @@
+#include "foo.h"
```

Comment:
`#include` get replaced in the same source file during preprocessing, so I am not sure what is the objective of splitting them.

It would make sense to test contracts within multiple translation units, but we are not doing that here. Same for all the other header files.

Reply:
Before, we added function contracts on function declarations and place them on `.h` files, which would make sense to test here. This is not the case anymore. We can test/add this feature on a separate PR. I remove all regression tests design for multiple files (from this PR).